server/views/components/customDisplay/crm4/crm4OfficeUseOnly.njk

```diff
@@ -0,0 +1,240 @@
+
+    {# DNA Report Section #}
+    {% if data.DnaReport %}
+        {% set tableDnaReport = {
+            head: [
+                { text: "Requested DNA Cost" },
+                { text: "Authorised DNA Cost" }
+            ],
+            rows: [
+                [
+                    { text: data.DnaReport.requested.dnaCost | formatCurrency },
+                    { text: data.DnaReport.authorised.dnaCost | formatCurrency }
+                ]
+            ]
+        } %}
+        {{ formSubsection('DNA Report', tableDnaReport, true) }}
+    {% endif %}
+
+    {# Accommodation Section #}
+    {% set fieldsAccommodation = [
+        { label: "Accommodation Cost Basis", value: data.Accommodation.costBasis }
+    ] %}
+    {{ formSubsection('Accommodation', fieldsAccommodation) }}
+
+    {# Transcription Section #}
+    {% set tableTranscription = {
+        head: [
+            { text: "" },
+            { text: "Number of Minutes" },
+            { text: "Cost Per Minute" },
+            { text: "Total Cost" }
+        ],
+        rows: [
+            [
+                { text: "Requested", isHeader: true },
+                { text: data.Transcription.requested.numMin },
+                { text: data.Transcription.requested.costPerMin | formatCurrency },
+                { text: data.Transcription.requested.total | formatCurrency }
+            ],
+            [
+                { text: "Allowed", isHeader: true },
+                { text: data.Transcription.authorised.numMin },
+                { text: data.Transcription.authorised.costPerMin | formatCurrency },
+                { text: data.Transcription.authorised.total | formatCurrency }
+            ]
+        ]
+    } %}
+    {{ formSubsection('Transcription', tableTranscription, true) }}
+
+    {# Photocopying Section #}
+    {% set tablePhotocopying = {
+        head: [
+            { text: "" },
+            { text: "Number of Pages" },
+            { text: "Cost Per Page" },
+            { text: "Total Cost" }
+        ],
+        rows: [
+            [
+                { text: "Requested", isHeader: true },
+                { text: data.Photocopying.requested.numPages },
+                { text: data.Photocopying.requested.costPerPage | formatCurrency },
+                { text: data.Photocopying.requested.total | formatCurrency }
+            ],
+            [
+                { text: "Allowed", isHeader: true },
+                { text: data.Photocopying.authorised.numPages },
+                { text: data.Photocopying.authorised.costPerPage | formatCurrency },
+                { text: data.Photocopying.authorised.total | formatCurrency }
+            ]
+        ]
+    } %}
+    {{ formSubsection('Photocopying', tablePhotocopying, true) }}
+
+    {# Authority Section #}
+    {% set fieldsAuthority = [
+        { label: "Amount Allowed", value: data.Authority.allowedAmount | formatCurrency },
+        { label: "Signed Authority", value: data.Authority.signedAuth }
+    ] %}
```

Comment:
**Destruction date** field is missing from Authority Section.

Reply:
added